test/testdata/infer/generics/type_params_redeclare.rb.autocorrects.exp

```diff
@@ -0,0 +1,43 @@
+# -- test/testdata/infer/generics/type_params_redeclare.rb --
+# typed: true
+
+# One-line format
+class MyAbstractClass
+  extend T::Sig, T::Helpers
+  abstract!
+
+  sig {abstract.type_parameters(:U).params(x: T.type_parameter(:U)).returns(T.type_parameter(:U))}
+  def example(x); end
+end
+
+class MyClass < MyAbstractClass
+# error: Missing definition for abstract method `MyAbstractClass#example` in `MyClass`
+  sig { override.type_parameters(:U).params(x: T.type_parameter(:U)).returns(T.type_parameter(:U)) }
+  def example(x); end
+end
+
+# Multi-line format
+class MultiParamAbstractClass
+  extend T::Sig, T::Helpers
+  abstract!
+
+  sig {abstract.type_parameters(:U).params(a: T.type_parameter(:U), b: Integer, c: String, d: Float, e: Symbol).returns(T.type_parameter(:U))}
+  def example(a, b, c, d, e); end
+end
+
+class MultiParamConcreteClass < MultiParamAbstractClass
+# error: Missing definition for abstract method `MultiParamAbstractClass#example` in `MultiParamConcreteClass`
+  sig do
+    override
+    .type_parameters(:U).params(
```

Comment:
Should the `.type_parameters` and `.params` be on separate lines?

That's what we do throughout the docs and stdlib types.